src/views/UnitsTable/UnitsTableView.tsx

```diff
@@ -169,4 +169,33 @@ const UnitsTableView: FunctionComponent<Props> = ({data, width, height}) => {
     )
 }
 
+export const sortIds = (ids: (string | number)[]) => {
+    // Handle cases:
+    //    1, 2, 3, ... These are numbers
+    //    A1, A2, A3, ..., A10, A11, B1, B2, B3, ... Note: A2 comes before A10
+    //    axyz, bxyz, bzzz, ... These are strings
+
+    return [...ids].sort((id1, id2) => {
+        if ((typeof(id1) === 'number') && (typeof(id2) === 'number')) {
+            return id1 - id2
+        }
+        else {
+            if ((!isNaN(parseInt(id1 + ''))) && (!isNaN(parseInt(id2 + '')))) {
+                return parseInt(id1 + '') - parseInt(id2 + '')
+            }
+            const firstChar1 = (id1 + '')[0]
+            const firstChar2 = (id2 + '')[0]
+            if (firstChar1 === firstChar2) {
+                const s1 = (id1 + '').slice(1)
+                const s2 = (id2 + '').slice(1)
+                if ((!isNaN(parseInt(s1))) && (!isNaN(parseInt(s2)))) {
+                    return parseInt(s1) - parseInt(s2)
+                }
+                else return id1 < id2 ? -1 : id1 > id2 ? 1 : 0
+            }
+            else return id1 < id2 ? -1 : id1 > id2 ? 1 : 0
+        }
+    })
+}
+
```

Comment:
Here's the new method of sorting unitIds. It handles the 3 cases shown in the comments. The second case is relevant for confusion matrix and comparisons between sortings

Reply:
This seems like a good idea--I don't know how strong the assumptions are that we can make about what people use for unit ids, clearly it bit me earlier when I assumed they'd be numbers.

Couple things that might be worth thinking about:

- `parseInt('123A') === parseInt('123B') === 123` so if we want to handle a case where there are suffixes, we might need to change the second case a bit.
- In the third case (same leading non-numeric character) it might be worthwhile to compare the strings in a loop or something to handle multiple-letter prefixes
- I guess if they have like 'A1' and '123' those would default to string comparison? That seems reasonable--it's not clear to me what an actual sort would be for them anyway